_docs/software_support_lifecycle.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Software Support Lifecycle
+date: 2020-12-09
+summary: Vox Pupuli policy on supporting upstream software
+---
+
+## Software Support Lifecycle
+
+As a part of Vox Pupuli's mission to provide a home for modules that still have a userbase and need development, we need to define a clear support strategy for what operating systems, dependencies, and upstream software we will support and for how long those will be supported.
+
+As we operate entirely on the graces of our community members being willing to voluntarily contribute to our modules, we also need to be respective of their time and resources and not add undue stress in order to support software that upstream creators no longer support.
```

Comment:
Should be “respectful” instead of “respective”.

Reply:
Oh my goodness, that is horrible and entirely on me.